src/main/java/seedu/address/logic/commands/queue/BreakCommand.java

```diff
@@ -17,7 +17,7 @@
     public static final String COMMAND_WORD = "break";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Doctor goes for a break\n"
+            + ": Doctor goes for a break. Marks the doctor as on break.\n"
```

Comment:
Rephrase to 'Marks the doctor as on break. Patients cannot be refered to the doctors on break.'